container.fc

```diff
@@ -105,6 +108,22 @@ HOME_DIR/\.local/share/containers/storage/volumes/[^/]*/.*	gen_context(system_u:
 /var/lib/docker-latest/overlay(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 /var/lib/docker-latest/overlay2(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 
+/var/lib/cni(/.*)?								gen_context(system_u:object_r:container_var_lib_t,s0)
+/var/lib/rancher/k3s(/.*)?							gen_context(system_u:object_r:container_var_lib_t,s0)
+/var/lib/rancher/k3s/data(/.*)?							gen_context(system_u:object_r:container_runtime_exec_t,s0)
+/var/lib/rancher/k3s/storage(/.*)?						gen_context(system_u:object_r:container_file_t,s0)
+#/var/lib/rancher/k3s/agent/containerd/[^/]*/snapshots(/.*)?			gen_context(system_u:object_r:container_share_t,s0)
```

Comment:
Why is this commented out?  Either we need it or we do not?

Reply:
I copied it from the original module, I'll drop it.